docs/concepts/packaging-types.adoc

```diff
@@ -53,5 +52,23 @@ executing the hollow JAR:
 ----
 $ java -jar myapp-hollow-swarm.jar myapp.war
 ----
-endif::[]
+
+=== Pre-Built hollow jars
```

Comment:
Sorry to be a pain, but when it comes to heading capitalization, maybe this should read `Pre-Built Hollow JARs`?